staging/src/k8s.io/api/testdata/HEAD/policy.v1.Eviction.json

```diff
@@ -53,6 +53,7 @@
     "propagationPolicy": "propagationPolicyValue",
     "dryRun": [
       "dryRunValue"
-    ]
+    ],
+    "ignoreStoreReadErrorWithClusterBreakingPotential": true
```

Comment:
interesting... should the eviction handler prevent setting this option to true in a pod evict request?

Reply:
I think the eviction handler should prevent it, maybe there is a better place to do it, i added the check inside `EvictionREST` https://github.com/kubernetes/kubernetes/pull/127513/commits/5e1885af6d47f460c9c48e8ebe1048b5f849038b

Reply:
Seems weird that the testdata here added it.  Have you worked out why yet?

Reply:
I thought the testdata adds a non default value for every field for api roundtrip testing, and eviction request has a field `deleteOptions *metav1.DeleteOptions`
https://github.com/kubernetes/kubernetes/blob/2bb886ce2a6ae5a83031efd225e8a17dd7ecba03/staging/src/k8s.io/api/policy/v1/types.go#L213-L223